poradnia/users/views.py

```diff
@@ -62,6 +63,15 @@ def get_object(self):
         # Only get the User record for the user making the request
         return Profile.objects.get_or_create(user=self.request.user)[0]
 
+    def get_form(self, *args, **kwargs):
+        # dynamically exclude event_reminder_time if user is not a staff member
+        if not self.request.user.is_staff:
+            exclude = ('event_reminder_time',)
+
+            self.form_class = modelform_factory(model=self.model, form=self.form_class, exclude=exclude)
```

Comment:
@ad-m faktycznie, użycie tego mixinu wydaje się dużo lepszym pomysłem, poprawię to w wolnej chwili